modules/sns-cronjob/README.md

```diff
@@ -0,0 +1,22 @@
+## Requirements
```

Comment:
have some samples, at least two. one with minimal configuration and one with complete configs